### ISSUE TITLE
Label on checkbox that enables/removes default tier, is misleading

### ISSUE THREAD

**Issue by Abekonge** [opencollective/opencollective#7077]
## Who is your user?

Collective admins

## TLDR
It is not clear how to remove the Custom Contribution Tier from the "edit tiers" page. The label of the checkbox is misleading and confusing. It is also not clear why it is not customizable

## What are they trying to achieve?

_When setting up my tiers, I want to remove or edit the default custom donation tier, so I can change the default tier or just remove it._

## How are they currently doing this?
 
Looking at the edit-tier page, I read that I cannot customize it, without understanding the reason for why. When looking to remove it - I would normally go into the tier by clicking `Edit tier` and then `Delete`. But as the Custom Contribution tier is not displayed as a tier UI-object, I cannot do this, leaving me confused.

The check box `Enable flexible contributions` removes the Custom Contribution tier when unchecked. But this is not at all understandable from the label. The Custom Contribution which is not customizable - is called "Custom Contribution", and the checkbox mentions "flexible contributions". So from the checkbox label I would expect it to remove the possibility to have "flexible amounts" in any tier. Not remove the default custom contribution.

Another thing is, that when the Custom Contribution tier is not displayed as a UI-object, there is no visible feedback as to whether it is there or not.

## How well understood is this problem?

 - [ ] Surfacing: if you feel this is a seed of an idea that needs to be researched.
 - [ ] Understanding: if you've done some research and feel it is already well understood and ready for shaping.
 - [x] Shaping:  if you feel a solution is clearly defined and can be made into a pitch ([read more about shaping](https://basecamp.com/shapeup/1.1-chapter-02))
 - [ ] Pitching: if you feel that it is ready to be presented for prioritisation ([read more about pitching](https://basecamp.com/shapeup/1.5-chapter-06))

## Possible solution 1 (very easy, not perfect)
Change the wording of the `Enable flexible contributions` checkbox to `Enable the default custom contribution tier`.

## Possible solution 2 (better)
The most important thing  (I feel) for understanding what is going on, is to add the default tier to the list of visible tier UI-objects, and make it editable (why shouldn't it be?). It supports the users understanding of what is going on by actually showing all the tiers. 

Then either you could remove the paragraph and checkbox entirely, removing the confusion. Or if you feel the info is important you could change it into 

> For many collectives it is a good idea to have a generic donation tier that doesn't enforce any minimum amount or interval. This makes it possible for people to donate in cases where they don't feel they fit into the custom tiers.

> [`Add default tier`]

The button `Add default tier`, when clicked, adds a default tier with the default words, to the list of the visible and editable tier UI-objects. That way people can always get it back if the remove it or delete the first one.

**Comment by Betree** [comment]
cc @opencollective/productdesign  
Related: #3055

**Comment by Abekonge** [comment]
> cc @opencollective/productdesign Related: #3055

If you want I could make a PR with a changed wording of the checkbox label, to e.g. `Enable default contribution tier`? I would like to try to setup a dev environment to be able to contribute, and this would be a simple issue to start with ...

**Comment by Betree** [comment]
@Abekonge We would need to update the documentation as well (https://docs.opencollective.com/help/collectives/collective-settings/tiers-goals), but yes, I think consolidating the wording to `Default contribution tier` would be clearer. 

I would also suggest showing the default tier's card in the grid below without the "Edit Tier" button (since it cannot be changed). Instead, we can show a small text like `The default contribution tier cannot be edited`.

About i18n: https://docs.opencollective.com/help/contributing/development/translations

**Comment by Abekonge** [comment]
@Betree I've updated the language-files. AND instead of putting the default contribution tier always in front, as you suggested, I've sorted the contribution tier cards according to the user-maintained sorting (the one used in the non-admin views of the contribution cards). Do you agree that it is better?
